examples/custom_provider/file_provider.c

```diff
@@ -0,0 +1,354 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+#define _GNU_SOURCE 1
+
+#include "umf/base.h"
```

Comment:
nit: please keep UMF headers after the system, also use <> as they should be included from the install dir

Reply:
Done